.vscode/settings.json

```diff
@@ -10,7 +10,7 @@
     "python.testing.pytestEnabled": true,
     "python.linting.flake8Enabled": false,
     "python.linting.pycodestyleEnabled": false,
-    "python.formatting.provider": "black",
+    "python.formatting.provider": "autopep8",
     "python.formatting.blackPath": "./env/bin/black", // "env\\Scripts\\black"
     "python.sortImports.args": [
         "-l",
```

Comment:
I guess this is dark red because the official JSON spec doesn't support comments. Also there's a trailing comma .

Reply:
Not sure if we want to keep these changes or if this was pushed accidentally (why would we change `pythonPath` or `formatting.provider`. I will revert them. @danixeee can check and update this file if necessary when he gets back.